lib/alchemy/config_loader.rb

```diff
@@ -0,0 +1,70 @@
+module Alchemy
+  class ConfigLoader
+    attr_reader :which
+
+    def initialize(which)
+      @which = which.sub(/\.yml$/, '')
+    end
+
+    # Reads the config file named +<which>.yml+ from +config/alchemy/+ folder.
+    #
+    def load_all
+      pths = paths
```

Comment:
Yes. Feels a bit misplaced though to cache only for the warning for "you have no YMLs".